.github/ISSUE_TEMPLATE/broken-website.md

```diff
@@ -36,12 +30,6 @@ Here is an example:
 - Notice that when hovering over the example button, it does not change color.
 -->
 
-## Expected Behavior
-<!-- Provide a clear and concise description of what you expected to happen. -->
```

Comment:
I'm not sure about this one, as I've seen a lot of bugs in the past where I was left wondering what the bug was as from my perspective it could be expected. Maybe reduce it to 1 section.

Reply:
Well, for broken websites I think this should be part of the steps - the last step in the example is essentially the unexpected observation.

Reply:
> Well, for broken websites I think this should be part of the steps

Well, the steps are 50% of the times skipped/useless. So these 2 sections + screenshot is most of the time the reliable information.